flowetl.Dockerfile

```diff
@@ -38,20 +43,23 @@ WORKDIR /${SOURCE_TREE}/flowetl
 
 COPY . /${SOURCE_TREE}/
 
-RUN apt-get update && \
+
+RUN     apt-get update && \
         apt-get install -y --no-install-recommends git build-essential && \
         pip install --no-cache-dir pipenv && pipenv install --clear --deploy --system && \
         apt-get -y remove git build-essential && \
         apt purge -y --auto-remove && \
         rm -rf /var/lib/apt/lists/*
+RUN echo $PWD && ls $PWD && ls $PWD/flowetl
 RUN apt-get update && \
         apt-get install -y --no-install-recommends git && \
         cd flowetl && \
         python setup.py install && \
         apt-get -y remove git && \
         apt purge -y --auto-remove && \
         rm -rf /var/lib/apt/lists/* && \
-        mv /${SOURCE_TREE}/flowetl/entrypoint.sh /
+        mv /${SOURCE_TREE}/flowetl/entrypoint.sh / && \
+	cp -r /${SOURCE_TREE}/flowetl/mounts/dags/ /opt/airflow/dags/
```

Comment:
Why's this here? (May become apparent as I work thru)

Reply:
Airflow goes looking for it's DAGs in /opt/airflow/dags by default - instead of messing with the Airflow config, I thought it better (closer to the airflow, one less configuration to mess about with) to copy the DAGs on build.

Reply:
The DAGs dir should be bind-mounted in so that users can define their own DAGs (and customise the standard DAGs if necessary) - as such, I don't think it can make sense to copy anything into the DAGs dir on build.

I don't think we should need to copy any flowetl internals into the DAGs dir, as long as users can import the required functions/classes from flowetl in their own DAG definitions. The DAGs defined in `flowetl/mounts/dags/` are useful for integration testing, and as a reference for defining production DAGs, but I think it's better if the actual python files that define the DAGs in a FlowKit deployment are user-supplied.